src/Pulp/Args/Types.purs

```diff
@@ -90,10 +90,12 @@ directories = {
   name: Just "<dir:dir:...>",
   parser: \arg -> do
     paths <- token <|> argErr arg "Needs a directory argument."
-    let paths' = split (Pattern Path.delimiter) paths
+    let paths' = ignoreNull $ split (Pattern Path.delimiter) paths
     for_ paths' requireDirectory
     pure $ Just (toForeign paths')
-  }
+  } where
+    ignoreNull [""] = []
```

Comment:
I guess it's due to my completely useless obsession with inaccurate micro-optimizations: Checking emptiness of head is O(1), calling filter is O(n).